### PR TITLE
Fix some audit comments

### PR DESCRIPTION
**Pull request by Loverush**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: cmd/geth/blsaccountcmd.go [hunk 1]
```diff
@@ -10,11 +10,11 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/ethereum/go-ethereum/signer/core"
```
Comment (NashBC) [inline cmd/geth/blsaccountcmd.go:13]: Move this to line 28.

File: params/config.go [hunk 2]
```diff
@@ -265,7 +265,7 @@ var (
 		BrunoBlock:          big.NewInt(13082000),
 
 		// TODO modify blockNumber, make sure the blockNumber is not an integer multiple of 200 (epoch number)
-		// TODO Caution !!! it should be very careful !!!
+		// Caution !!! it should be very careful !!!
```
Comment (NashBC) [inline params/config.go:268]: Keep TODO in this PR, once we have the upgrade plan, we can propose another PR to modify this upgrade height.

File: eth/filters/filter_system.go [hunk 3]
```diff
@@ -153,7 +153,7 @@ func NewEventSystem(backend Backend, lightMode bool) *EventSystem {
 	if m.txsSub == nil || m.logsSub == nil || m.rmLogsSub == nil || m.chainSub == nil || m.pendingLogsSub == nil {
 		log.Crit("Subscribe for event system failed")
 	}
-	if m.voteSub == nil {
+	if m.voteSub == nil || m.finalizedHeaderSub == nil {
 		log.Warn("Subscribe for vote event failed")
```
Comment (NashBC) [inline eth/filters/filter_system.go:157]: The log should be log.Warn("Subscribe for vote or finalized header event failed")

File: cmd/geth/blsaccountcmd.go (hunk 1 above)
Comment (Loverush) [reply cmd/geth/blsaccountcmd.go:13]: Done

File: params/config.go (hunk 2 above)
Comment (Loverush) [reply params/config.go:268]: Fixed

File: eth/filters/filter_system.go (hunk 3 above)
Comment (Loverush) [reply eth/filters/filter_system.go:157]: Fixed

**Pull request merged**